### DISCUSSION TITLE
Support GitHub project

### DISCUSSION THREAD

**Discussion by ryanxingql** [community/community#43925] [Mobile] [answered]
Hi
I would like to check my projects with mobile.
Is there any plan to support it, thank you

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.94.0 (86495482)
Device: iPhone 13 Pro (iOS 16.2)
</details>

**Comment by alcere** [comment, marked as answer]
Hi @ryanxingql, yes the team is currently working adding support for projects! 
The feature has been shipped to Beta users and it is now under development polishing the experience, stay tunned for updates!

**Reply by ryanxingql** [reply]
Thank you so much
